dlt/destinations/impl/lancedb/lancedb_client.py

```diff
@@ -731,6 +720,19 @@ def run(self) -> None:
         with FileStorage.open_zipsafe_ro(self._file_path) as f:
             records: List[DictStrAny] = [json.loads(line) for line in f]
 
+        # Replace empty strings with placeholder string if OpenAI is used.
```

Comment:
Can't tell the impact on performance, but I think it's a good fix until there's progress on the LanceDB issue! 

I don't know how frequently you'd hit an empty string when embedding, but it might be worth mentioning in the docs?

Reply:
@Pipboyguy didn't we switch the format to parquet? I think it is in PR that is still in review. anyway we'll be able to use `pa.compute`  to replace those soon

Reply:
@rudolfix yes indeed, it does make it a bit tricky to implement a fix considering the switch in format.

Reply:
@zilto agreed, will add a doc entry for this!